storage/client_event_test.go

```diff
@@ -33,12 +35,14 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+const numberOfProtoMethods = 30
```

Comment:
Where did this come from? If we need a constant for this it should come from the proto package where the method enum is defined.